src/pat/modal/tests.js

```diff
@@ -55,6 +57,7 @@ define(["pat-modal"], function(pattern) {
                 expect($modal.find(".header .close-panel").length).toBeTruthy();
                 expect($modal.find(".panel-content p").length).toBe(2);
                 expect($modal.find(".panel-content h4").length).toBe(1);
+                expect($modal.hasClass("modal-active")).toBeTruthy();
             });
 
             it("Modal with a form that has the pat-modal CSS class", function() {
```

Comment:
Is this related to my request to Alexander to have a class modal-active on body when a modal is active? It's not useful on the modal container anyway. 

> Op 17 okt. 2015 om 09:47 heeft JC Brand <notifications@github.com> het volgende geschreven:
> 
> In src/pat/modal/tests.js:
> 
> > @@ -55,6 +57,7 @@ define(["pat-modal"], function(pattern) {
> >                  expect($modal.find(".header .close-panel").length).toBeTruthy();
> >                  expect($modal.find(".panel-content p").length).toBe(2);
> >                  expect($modal.find(".panel-content h4").length).toBe(1);
> > +                expect($modal.hasClass("modal-active")).toBeTruthy();
> >              });
> >  
> >              it("Modal with a form that has the pat-modal CSS class", function() {
> Please add a test for the class parser argument to check that by default modal-active is set but that it can be changed to other values.
> 
> —
> Reply to this email directly or view it on GitHub.
>